Description: Exclude vendor directory from workspace
 We want to use the dh-cargo-vendored-source script, but it needs the vendored
 dependencies not to believe they're in a workspace. This isn't an issue in typical Rust
 packages as they're usually based off the crates.io tarball, but this is far from a
 typical package.
Author: Simon Chopin <schopin@ubuntu.com>
Origin: ubuntu
Forwarded: not-needed
Last-Update: 2022-08-25
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -42,6 +42,9 @@
   "src/tools/x",
   # stdarch has its own Cargo workspace
   "library/stdarch",
+  # dh-cargo-vendored-sources needs to be able to run cargo metadata on
+  # vendored crates
+  "vendor/",
 ]
 
 [profile.release.package.compiler_builtins]
